wordpress-mobile/WordPress-iOS
# Jetpack Focus: Remove footnote in new users overlay

Ref: p1676550754200889-slack-C04564QS0V7

## Description
This PR fixes an issue with the New Users Overlay where “Switching is free and only takes a minute” was being displayed twice.

## Testing Instructions

Test on both iPhone and iPad:

1. Run the app
2. Open the debug menu and enable the "Jetpack Features Removal Phase For New Users" flag
3. Close & re-open the app
4. An overlay should appear
5. Make sure the footnote "Switching is free and only takes a minute" is not displayed and only the subtitle is displayed.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
